### PR TITLE
Enabling testsuite/integration/microprofile-tck/config to run against…

### PR DESCRIPTION
**Pull request by fabiobrz**
```
$ mvn clean test -Dts.bootable -Dmaven.repo.local=/home/fburzigo/projects/git/wildfly/wildfly/local-repo -DtestLogToFile=true -pl config/ -am
...
[INFO] Tests run: 168, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 22.568 s - in TestSuite
[INFO] 
[INFO] Results:
[INFO] 
[INFO] Tests run: 168, Failures: 0, Errors: 0, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for WildFly Test Suite: Integration - MicroProfile TCK 21.0.0.Beta1-SNAPSHOT:
[INFO] 
[INFO] WildFly Test Suite: Integration - MicroProfile TCK . SUCCESS [  2.463 s]
[INFO] WildFly Test Suite: Integration - MicroProfile TCK - Config SUCCESS [ 32.488 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  35.940 s
[INFO] Finished at: 2020-08-20T17:12:27+02:00
[INFO] ------------------------------------------------------------------------
```

### CODE REVIEW

**Comment by fabiobrz** [comment]
Hi @marekkopecky - I've done this for config and was wondering whether you could have a quick look. It follows the same pattern used for instance by https://github.com/fabiobrz/wildfly/pull/4 or https://github.com/fabiobrz/wildfly/pull/2, hence reflecting the `layers.profile` profile configuration.

File: testsuite/integration/microprofile-tck/config/pom.xml [hunk 1]
```diff
@@ -183,5 +183,112 @@
...
+                                </goals>
+                                <phase>none</phase>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.jboss.galleon</groupId>
+                        <artifactId>galleon-maven-plugin</artifactId>
+                        <executions>
+                            <!-- Disable the Galleon provisioning -->
```
Comment (marekkopecky) [inline testsuite/integration/microprofile-tck/config/pom.xml]: @fabiobrz Can you please double check this definition of galleon-maven-plugin?

It seems that galleon-maven-plugin is defined in layers.profile only. So galleon-maven-plugin is not activated by default in bootablejar.profile, so it is not necessary to disable it in bootablejar.profile.

I also checked this profile locally and it seems that this profile works correctly without this galleon-maven-plugin disabling.

**Comment by marekkopecky** [comment]
@fabiobrz Do you have CI ready for this MR?

File: testsuite/integration/microprofile-tck/config/pom.xml [hunk 2]
```diff
@@ -183,5 +183,112 @@
...
+            <activation>
+                <property>
+                    <name>ts.bootable</name>
+                </property>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>org.wildfly.arquillian</groupId>
+                    <artifactId>wildfly-arquillian-container-bootable</artifactId>
+                    <scope>test</scope>
```
Comment (marekkopecky) [inline testsuite/integration/microprofile-tck/config/pom.xml]: Wouldn't be dependency-management the better place for scope definition? If the scope is defined in dependency-management, than it's not necessary to define it in all modules

Comment (fabiobrz) [reply testsuite/integration/microprofile-tck/config/pom.xml]: Thanks, on it right now.

File: testsuite/integration/microprofile-tck/config/pom.xml (hunk 1 above)
Comment (fabiobrz) [reply testsuite/integration/microprofile-tck/config/pom.xml]: Thanks, looking into it

File: testsuite/integration/microprofile-tck/config/pom.xml (hunk 2 above)
Comment (fabiobrz) [reply testsuite/integration/microprofile-tck/config/pom.xml]: Actually see that here [1] it is not defined and delegated to the single module in some cases, e.g.: https://github.com/wildfly/wildfly/blob/master/testsuite/integration/elytron/pom.xml#L133
WDYT?

[1]
https://github.com/wildfly/wildfly/blob/master/pom.xml#L8624

Comment (marekkopecky) [reply testsuite/integration/microprofile-tck/config/pom.xml]: Ok, scope is not defined in dependency-management at wildfly-arquillian-container-managed. But still, wouldn't be better to do it and the same for wildfly-arquillian-container-bootable dependency?

Anyway, this PR don't have to fix wildfly-arquillian-container-managed, but this PR should define wildfly-arquillian-container-bootable in clean way. WDYT?

Comment (fabiobrz) [reply testsuite/integration/microprofile-tck/config/pom.xml]: You're right. It is used for tests and that's all. I agree and will fix it.

**Comment by marekkopecky** [comment]
@fabiobrz Thank you for updates. PR LGTM, although I don't have a permission to merge it. Anyway feel free to merge this PR yourself

**Pull request merged**